paddle/framework/op_registry_test.cc

```diff
@@ -36,8 +36,9 @@ class MyTestOpProtoAndCheckerMaker : public OpProtoAndCheckerMaker {
  public:
   MyTestOpProtoAndCheckerMaker(OpProto* proto, OpAttrChecker* op_checker)
       : OpProtoAndCheckerMaker(proto, op_checker) {
-    AddInput("input", "input of cosine op");
-    AddOutput("output", "output of cosine op");
+    AddInput("input", "input of cosine op", /*multiple*/ true);
```

Comment:
offer a `AddInputs`  to repalce `AddInput("input", true)` ? 

two methods to add inputs:
- `AddInput` to add a normal input
- `AddInputs` to add a variable length input